### PR TITLE
404 not found challenge solved

### PR DESCRIPTION
**Pull request by susird**
This is my pr for the amazing 404 not found challenge :heartbeat:

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
```
Comment (josepplloo) [inline index.html:2]: 💅 https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/lang

File: index.html [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  <div class=" footer">
+    <div class="credits">
+      <p class="credit-creator">created by <strong>username</strong>- devChallenges.io</p>
+    </div>
+  </div>
+
+
+</body>
+
+</html>
```
Comment (josepplloo) [inline index.html:45]: A blank line is needed at the bottom of this file to avoid compilation errors in some tools.

File: style.css [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+    margin-left: 740px;
+    margin-top: 663px;
+  }
+
+  .credit-creator { 
+    margin-left: 577px;
+    margin-top: 1039px;
+  }
+  
+  }
```
Comment (josepplloo) [inline style.css:145]: idem, blank line

**Review by josepplloo — CHANGES_REQUESTED** [review]
Very good job, you deserve everything! I have some recommendations for you to do the best! what do you prefer: 
1st option, I can write all the comments meanwhile you solve them straight away. It could take some rounds of comments - solve.
2nd option, Talk whit you about the recommendations and after that, you can solve them. finally, if you omit something I will put a comment.
the changes are about, missing html5 tags, unused classes, and missing/excess styles

File: index.html (hunk 1 above)
Comment (susird) [reply index.html:2]: What kind of change do I need to make here?

**Comment by susird** [comment]
> Very good job, you deserve everything! I have some recommendations for you to do the best! what do you prefer:
> 1st option, I can write all the comments meanwhile you solve them straight away. It could take some rounds of comments - solve.
> 2nd option, Talk whit you about the recommendations and after that, you can solve them. finally, if you omit something I will put a comment.
> the changes are about, missing html5 tags, unused classes, and missing/excess styles

File: index.html (hunk 1 above)
Comment (josepplloo) [reply index.html:2]: it is a good comment

**Pull request closed without merging**